tsa-core/src/main/kotlin/org/usvm/machine/interpreter/TvmDictOperationInterpreter.kt

```diff
@@ -1383,33 +1506,79 @@ class TvmDictOperationInterpreter(
         val dictId = DictId(keyLength)
         val resultDict = scope.calcOnState { memory.allocConcrete(TvmDictCellType) }
 
+        val keySort = ctx.mkBvSort(keyLength.toUInt())
+        val isInput = checkDictIsInput(scope, dictCellRef, dictId, keySort)
+        if (isInput) {
+            doSetInstOnInputDict(
+                scope = scope,
+                getOldValue = getOldValue,
+                dictCellRef = dictCellRef,
+                keySort = keySort,
+                keyKind = keyKind,
+                mode = mode,
+                key = key,
+                resultDict = resultDict,
```

Comment:
Also, `resultDict` can be calculated inside the function to reduce number of arguments